
---
int32 width
int32 height
int32 channels
